invoicing/functions/server/publicApi.js

```diff
@@ -0,0 +1,27 @@
+/* This code is under Copyright (C) 2019 of Cegid SAS all rights reserved */

```

Comment:
We don't have copyright in other files, I think we can removes it here as well.

Reply:
Good shout, done. Thanks!